lib/email/receiver.rb

```diff
@@ -900,15 +901,42 @@ def add_attachments(raw, user_id, options = {})
             else
               raw << "\n\n#{attachment_markdown(upload)}\n\n"
             end
+          else
+            rejected_attachments << upload
+            raw << "\n\n#{I18n.t('emails.incoming.missing_attachment', filename: upload.original_filename)}\n\n"
           end
         ensure
           tmp&.close!
         end
       end
+      notify_about_rejected_attachment(rejected_attachments) if rejected_attachments.present?
 
       raw
     end
 
+    def notify_about_rejected_attachment(attachments)
+      errors = []
+
+      attachments.each do |a|
+        error = ''
+        a.errors.messages.each do |attr, msg|
+          error = msg.first
+        end
```

Comment:
This looks weird. Why do a loop over all `messages` when it uses only one error message?